mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -557,8 +598,33 @@
         <Arc value="0"/>
         <Range index="0" value="5"/>
     </CAbilEffectTarget>
+    <CAbilMorph id="LiberatorMorphtoAA">
+        <InfoArray index="0">
+            <SectionArray index="Abils">
+                <DurationArray index="Delay" value="0"/>
+            </SectionArray>
+            <SectionArray index="Mover">
+                <DurationArray index="Duration" value="0"/>
+            </SectionArray>
+            <SectionArray index="Stats">
+                <DurationArray index="Delay" value="0"/>
```

Comment:
Fixed an issue where Liberators could not be given certain orders immediately after being ordered to unsiege
- Unsieging Liberators had the stats transformation delay reduced from 1.5417 to 0.0

I assume that this is necessary for certain orders like attacking air units since the anti-ground version has no anti-air weapon.
This should also cause the info panel and wireframe to change immediately.
There is a chance that this could break the animation or death model. This should be tested to verify that all looks fine